docs/src/main/sphinx/security/internal-communication.rst

```diff
@@ -25,107 +26,36 @@ command:
 Internal SSL/TLS configuration
 ------------------------------
 
-SSL/TLS is configured in the ``config.properties`` file.  The SSL/TLS on the
-worker and coordinator nodes are configured using the same set of properties.
-Every node in the cluster must be configured. Nodes that have not been
-configured, or are configured incorrectly, are not able to communicate with
-other nodes in the cluster.
+The SSL/TLS on the worker and coordinator nodes are configured using the same
+set of properties. Every node in the cluster must be configured. Nodes that have
+not been configured, or are configured incorrectly, are not able to communicate
+with other nodes in the cluster.
 
-To enable SSL/TLS for Trino internal communication, do the following:
+You must :doc:`/security/tls` for access to the cluster, before enabling SSL/TLS
+for Trino internal communication between the coordinator and all workers. Ensure
+the following configuration is identical on all cluster nodes.
 
-1. Disable HTTP endpoint.
+1. Configure a shared secret for internal communication as described in
+   the preceding section.
 
-   .. code-block:: text
+2. Set a name for the cluster environment in ``etc/node.properties``:
```

Comment:
I'm pretty sure this is already required for the customer.

Reply:
So leave it out?

Reply:
Removed..

Reply:
yes leave it out